assets/scss/templates/_header.scss

```diff
@@ -234,45 +231,46 @@
     top: -7px;
   }
 
+  &.open {
+    visibility: visible;
+    opacity: 1;
+  }
+
   header {
     display: flex;
     align-items: center;
-  }
 
-  span {
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    max-width: 248px;
-    display: block;
-    overflow: hidden;
-  }
+    p {
+      margin-left: 10px;
 
-  header p {
-    margin-left: 10px;
-  }
+      span {
+        @include do-ellipsis(248px, block);
+      }
+    }
 
-  header .email {
-    color: $dp-color-grey;
-    font-size: $dp-font-size-small;
-  }
+    .email {
+      color: $dp-color-grey;
+      font-size: $dp-font-size-small;
+    }
 
-  .name {
-    color: $dp-color-darkgrey;
-    font-size: 20px;
+    .name {
+      color: $dp-color-darkgrey;
+      font-size: 20px;
```

Comment:
I think we were using variables for font-size would you mind checking this? Same for `font-size: 11px;` in  `.option-user li`

Reply:
@cbernat I did not refactor any of the styles. Just cleaned up indentation and fixed the bug. I guess we should do some refactoring once everything is defined.